modules/features2d/include/opencv2/features2d.hpp

```diff
@@ -529,11 +531,16 @@ class CV_EXPORTS_W GFTTDetector : public Feature2D
     CV_WRAP virtual void setBlockSize(int blockSize) = 0;
     CV_WRAP virtual int getBlockSize() const = 0;
 
+    CV_WRAP virtual void setGradientSize(int gradientSize);
+    CV_WRAP virtual int getGradientSize();
```

Comment:
As it's said before, new virtual methods break ABI.
You need to fix this as it's described [earlier](https://github.com/opencv/opencv/pull/9618#discussion_r138858648)